{{cookiecutter.repo_name}}/environment.yaml

```diff
@@ -4,7 +4,8 @@ channels:
 dependencies:
     - black=23.1.0
     - nbqa=1.6.1
-    - ipykernel=6.21.1
+    {% if cookiecutter.install_jupyter == 'yes' %}- jupyter=1.0.0{% endif %}
```

Comment:
chosing 'no' for jupyter will leave an empty new line in the environment.yaml
please change this using, e.g., jinja whitespace control
https://jinja.palletsprojects.com/en/3.0.x/templates/#whitespace-control

Reply:
Done!